src/java.base/share/classes/jdk/internal/access/JavaLangAccess.java

```diff
@@ -333,6 +333,45 @@ public interface JavaLangAccess {
      */
     byte[] getBytesNoRepl(String s, Charset cs) throws CharacterCodingException;
 
+    /**
+     * Returns a new string by decoding from the given latin1 bytes array.
+     *
+     * The caller of this method shall relinquish and transfer the ownership of
+     * the byte array to the callee since the later will not make a copy.
+     *
+     * @param bytes the byte array source
+     * @return the newly created string
+     * @throws IllegalArgumentException for malformed or unmappable bytes.
+     */
+    String newStringLatin1NoRepl(byte[] bytes);
+
+    /**
+     * Returns the string representation size for a given long value.
+     *
+     * @param x long value
+     * @return string size
+     *
+     */
+    int stringSize(long i);
+
+    /**
+     * Pack the two ascii characters corresponding to the value from 0 to 100 into a short
```

Comment:
Wording and name of this method can be improved. Suggestion:
`short digitPair(int i)`
"For values from 0 to 99 return a `short` encoding a pair of ASCII-encoded digit characters in little-endian, e.g. `0 -> ('0' << 8 | '0')`. Used for formatting."